app/routes/overview/components/Overview.css

```diff
@@ -4,7 +4,7 @@
   display: inline-block;
   white-space: nowrap;
   vertical-align: middle;
-  overflow: auto;
+  overflow: hidden;
```

Comment:
I notice this is the commit from the other pull request.
To avoid this remember to go via master when creating a new branch.

If can go smoothly if the other branch is merged directly followed by this, but in most cases, you will have to rebase.

And if this is merged before the other branch there will be some conflicts.